### ISSUE TITLE
How's the training result?

### ISSUE THREAD

**Issue by AnjieCheng** [nyoki-mtl/pytorch-segmentation#8]
Thank you for sharing your codes!

Is there any other config that can reproduce the paper result, since I stuck at best valid iou 0.47 when I tried to train it on on Cityscapes using DeepLabV3+(default config)?

Thanks again and appreciate your help!

**Comment by nyoki-mtl** [comment]
@AnjieZheng 
I didn't train any model on Cityscapes, so I don't have the training result.
But I confirmed that this code can make the significant result because I won 2nd prize on this conptetition (https://signate.jp/competitions/143) using DeepLabV3+.

When I trained the model on the dataset, I used pretrained model on cityscapes.
This is very important because BatchSize=2 makes batch normalization training unstable.
So, If you want to train DeepLabV3+ on Cityscapes from initial weights, I recommend you to increase the batchsize by using multi-gpu or lowering image resolution.

**Comment by cuevhv** [comment]
> @AnjieZheng
> I didn't train any model on Cityscapes, so I don't have the training result.
> But I confirmed that this code can make the significant result because I won 2nd prize on this conptetition (https://signate.jp/competitions/143) using DeepLabV3+.
> 
> When I trained the model on the dataset, I used pretrained model on cityscapes.
> This is very important because BatchSize=2 makes batch normalization training unstable.
> So, If you want to train DeepLabV3+ on Cityscapes from initial weights, I recommend you to increase the batchsize by using multi-gpu or lowering image resolution.

Did you use freeze_bn function (freeze the batch normalization) to train your network for the competition? What was your batch size and image size?

Thanks
